lib/vanagon/project.rb

```diff
@@ -101,6 +101,22 @@ def get_replaces
       replaces.flatten.uniq
     end
 
+    # Grabs a specific service based on which name is passed in
+    # note that if the name is wrong or there was no
+    # @component.install_service call in the component, this
+    # will return nil
+    #
+    # @param [string] name of service to grab
+    # @return [@component.service obj] specific service
+    def get_service(name)
+      @components.each do |component|
+        if component.name == name
```

Comment:
Yeah, you can definitely make services without names. But not the user, someone who's editing vanagon. If this method is called on a service object that doesn't have `name` for some reason, it's going to fail with `undefined method `name'`. I'm thinking that's on us, and this is fine as is.